legend-engine-xt-documentStore-pure/src/main/resources/core_documentstore/pureToDocumentStoreQuery/pureToDocumentStoreQuery.pure

```diff
@@ -1046,6 +1095,62 @@ function meta::external::store::document::functions::pureToDocumentStoreQuery::b
    $res;
 }
 
+function meta::external::store::document::functions::pureToDocumentStoreQuery::processMap(f:FunctionExpression[1], currentPropertyMapping:PropertyMapping[*], operation:SelectDocumentQuery[1], vars:Map<VariableExpression, ValueSpecification>[1], state:State[1], context:DebugContext[1], extensions:Extension[*]):OperationWithParentPropertyMapping[1]
```

Comment:
this function will be removed

Reply:
can wait for next MR, as long as the build doesn't break